app/src/main/java/org/p2p/wallet/auth/ui/phone/PhoneNumberEnterFragment.kt

```diff
@@ -75,12 +100,21 @@ class PhoneNumberEnterFragment :
     }
 
     override fun navigateToSmsInput() {
-        replaceFragment(NewSmsInputFragment.create())
+        replaceFragment(
+            NewSmsInputFragment.create(
+                countryCode,
+                binding.editTextPhoneNumber.text.toString()
```

Comment:
you can extend PhoneNumber class and add method getText() to avoid using text.toString() every time, when you need String

Reply:
named args please